ulb.py

```diff
@@ -62,13 +77,37 @@ def ip_parser(s):
 # Get configuration from Arguments
 ifnet = args.ifnet                   # network interface to attach xdp program
 virtual_server = args.virtual_server # virtual server (ethernet and IP address)
-real_servers = args.real_server      # list of real servers (ethernet and IP address)
 ports = args.port                    # ports to load balance
 debug = args.debug                   # bpf verbosity
 
-print("\nLoad balancing UDP traffic over {} interface for port(s) {} from :".format(ifnet, ports, ip_ntostr(virtual_server['ip'])))
-for real_server in real_servers:
-    print ("VIP:{} <=======> Real Server:{}".format(server_tostr(virtual_server), server_tostr(real_server)))
+real_servers = []                    # list of real servers
+config_file = args.config_file       # config file containing real servers list
+config_file_mtime = 0                # last modification time of config file
+
+def load_config(cfgFile):
+    """Load configuration from file object and return a list of server"""
+    print ("\nLoading real servers from {} file ...".format(cfgFile.name))
+    config = configparser.ConfigParser(allow_no_value=True)
+    config.read_file(cfgFile)
+    rs = []
+    for ip in config["Real Servers"]:
+        rs.append(ip_parser(ip))
```

Comment:
What happens if ip_parser() fails? Is the error ignored?

Reply:
exception is raised and will be catched by both `load_config` callers :+1: 
 - One at start-up and the script is stopped.
 - the other, when config change dynamically, in this case if we can loadconfig we just keep the previous one.